docs/getting-started/csharp.md

```diff
@@ -122,6 +122,30 @@ namespace MyCompany.MyApp
 }
 ```
 
+## Using Terraform Providers
+
+As you can see in the example above we want to deploy AWS resources, therefore we need to use the [Terraform AWS Provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs). The default way to work with a provider is to specify it in your local `cdktf.json` file, like this:
```

Comment:
If this is really the default way, then I think we should change the templates back to include a provider (maybe even one chosen during init flow).

However, I actually believe that using pre-built providers will be the common case. Using just a major cloud provider and a couple auxiliary (random, null) providers will cover a large amount of workflows. In this case, we should really have the examples using pre-built providers, but that hinges some on #558.

I do like being more explicit about to specify modules/providers though.

Reply:
Maybe default is the wrong wording here 🤔 I think a lot of people are confused on what providers / modules can be used. This is why I wanted to say default since all providers work this way. But I think you are totally right, a lot of use cases can be covered with prebuilt providers alone, so maybe I just say one way and mention it as another way 🤔

Reply:
Yes, I'd default to pre-built providers and use customly built providers if there are either special version requirements for the provider, or the provider is not available (yet) as a pre-built provider.

Reply:
I think I got this now, I made clear that every provider and module can be used while avoiding default entirely :)